plugins/GSdx/GSdx.rc

```diff
@@ -143,6 +143,7 @@ BEGIN
     COMBOBOX        IDC_SPRITEHACK,88,158,58,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     CONTROL         "Fast Texture Invalidation",IDC_FAST_TC_INV,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,102,92,8
     CONTROL         "Auto Flush", IDC_AUTO_FLUSH,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,128,70,8
+    CONTROL         "Half-pixel Offset New", IDC_OFFSETHACK_NEW,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,138,92,8
```

Comment:
There shouldn't be a space between `"Half-pixel Offset New",` and `IDC_OFFSETHACK_NEW`. Same goes for the Auto Flush line above. You can fix it that too it you feel like it. ;)